transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -195,6 +195,16 @@ interface MessagingClient {
     @Throws(IllegalStateException::class)
     fun detach(attachmentId: String)
 
+    /**
+     * Refresh the downloadUrl.
```

Comment:
Do we need to add here explanation of why we need this? I mean, an explanation about when the URL expires and such

Reply:
It is a good point. I will add a sentence about it .